rocAL/rocAL/rocAL_hip/box_encoder_hip.cpp

```diff
@@ -239,7 +239,7 @@ void BoxEncoderGpu::Run(pMetaDataBatch full_batch_meta_data, float *encoded_boxe
     int total_num_boxes = 0;
     for (int i = 0; i < _cur_batch_size; i++) {
```

Comment:
please remove all the debug print statements form this file

Reply:
Done